content/applications/finance/payment_acquirers.rst

```diff
@@ -81,7 +83,8 @@ Online Payment Providers
 +-------------------------------------+----------------------+-----------+------------+-----------------+
 | SIPS                                | ✔                    |           |            |                 |
 +-------------------------------------+----------------------+-----------+------------+-----------------+
-| Stripe                              | ✔                    | ✔         | ✔          |                 |
+| :doc:`Stripe                        | ✔                    | ✔         | ✔          |                 |
+| <payment_acquirers/stripe>`         |                      |           |            |                 |
```

Comment:
Why is this change needed for the Adyen migration to HPP?

Reply:
My intention was to add a link to the doc of the acquirer (if it exists in this version). But I see that I spoil a bit of master here :3

Reply:
If you want to revamp the doc en V13.0, please do so!!! =D

... But maybe not with a task called "Migrate Adyen to HPP" :smile: